common/app/components/VideoPlayer.tsx

```diff
@@ -1025,6 +1025,22 @@ export default function VideoPlayer({
         );
     }, [keyBinder, playerChannel]);
 
+    useEffect(() => {
+        const unbindRepeat = keyBinder.bindToggleRepeat(
+            (event) => {
+                event.preventDefault();
+                if (playMode === PlayMode.repeat) {
```

Comment:
You'll want to use `togglePlayMode` instead, otherwise the keyboard shortcut won't appear to work when the `VideoPlayer`'s `iframe` is in focus.